services/agents/mysql_test.go

```diff
@@ -40,7 +41,10 @@ func TestMySQLdExporterConfig(t *testing.T) {
 		Password:      pointer.ToString("s3cur3 p@$$w0r4."),
 		AgentPassword: pointer.ToString("agent-password"),
 	}
-	actual := mysqldExporterConfig(mysql, exporter, redactSecrets)
+	// TODO: Populate version information
+	agentVersion := &version.Parsed{}
```

Comment:
here too, let's use pmm
```suggestion
	pmmAgentVersion := &version.Parsed{}
```

Reply:
let's set current pmm-agent version in one test and old pmm-agent version in the another one.

Reply:
Done